### PR TITLE
no script stuff

### PR DESCRIPTION
**Pull request by ghost**
So that visitors without javascript enabled don’t have buttons that don’t work (About Me and Projects). Also added a blurb telling them that using js-enabled browsers would be better

### CODE REVIEW

**Comment by dhong44** [comment]
Thanks for the contribution! I just don't think this is necessary really. Some data showing the number of users that don't have JavaScript enabled would really help. The data could be for this site specifically or t he web in general. I know many more popular sites do not even allow users to use them unless JavaScript is enabled so this type of guard seems irrelevant in the modern world.

**Comment by ghost** [comment]
Let’s discuss offline but there still remain around .2% of internet traffic who disable Js. While this may seem small, anything at the internet's scale makes one in a billion occurrences quite common actually. I was confused why the about me and projects options were kept available when they were non functional without Js.

File: index.html [hunk 1]
```diff
@@ -20,7 +20,8 @@
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/skeleton.css">
   <link rel="stylesheet" href="css/style.css">
-
+  <noscript>
+    <link rel="stylesheet" href="css/noscript.css"></noscript>
```
Comment (cshen98) [inline index.html]: put this either all on the same line, or put the end tag at the same indentation level as the start

File: index.html [hunk 2]
```diff
@@ -66,14 +67,23 @@ <h6 class="menu-item">
               </h6>
             </div>
           </div>
+          <div class="row">
+            <noscript>
+              <br />
+              <center>Hi! This site is best viewed with javascript enabled! To learn more about me, as well as some
```
Comment (cshen98) [inline index.html]: Change this to just: "This site is best viewed with javascript enabled."

Comment (cshen98) [inline index.html]: don't use center tags, use css align instead

File: index.html [hunk 3]
```diff
@@ -66,14 +67,23 @@ <h6 class="menu-item">
               </h6>
             </div>
           </div>
+          <div class="row">
+            <noscript>
+              <br />
```
Comment (cshen98) [inline index.html]: don't use br here, use css spacing instead

File: index.html [hunk 4]
```diff
@@ -66,14 +67,23 @@ <h6 class="menu-item">
               </h6>
             </div>
           </div>
+          <div class="row">
```
Comment (cshen98) [inline index.html]: put this div inside the noscript i think

File: index.html (hunk 1 above)
Comment (cshen98) [inline index.html]: also is there a way to do this without using a separate css file, can we define onlywithjs in style.css and then add in something like a media query that checks whether js is enabled

**Comment by dhong44** [comment]
image

This is an example of a website that does not allow users that have not enabled JavaScript. @cshen98's website is not on this scale, if they can afford to not support these users then I believe @cshen98 can as well. I think instead simply suggesting users enable JavaScript, we should be blocking access to the site without it.

File: index.html [hunk 5]
```diff
@@ -66,14 +67,23 @@ <h6 class="menu-item">
...
+            <noscript>
+              <br />
+              <center>Hi! This site is best viewed with javascript enabled! To learn more about me, as well as some
+                projects I've been working on, please use a javascript-enabled browser.</center>
```
Comment (cshen98) [inline index.html]: dont add extra line spaces in text

File: index.html [hunk 6]
```diff
@@ -66,14 +67,23 @@ <h6 class="menu-item">
...
         </div>
       </div>
       <div class="animated not-active" id="aboutme_content-wrapper">
         <div class="container">
           <div class="row">
             <div class="twelve columns">
-              <p>Hi! I'm a software engineering student at the University of Waterloo and am currently looking for internship opportunities for the summer (May to Aug) 2019 term.</p>
-              <p>My experience is mainly in backend and data engineering disciplines, but I've also been exposed to distributed computing and machine learning technologies/systems.</p>
+              <p>Hi! I'm a software engineering student at the University of Waterloo and am currently looking for
+                internship opportunities for the summer (May to Aug) 2019 term.</p>
```
Comment (cshen98) [inline index.html]: see above about not adding extra line spaces in text

File: index.html [hunk 7]
```diff
@@ -84,15 +94,17 @@ <h6 class="menu-item">
         <div class="container">
           <div class="row">
             <div class="twelve columns">
-              <p>I'm hoping to be able to share some projects here that I've worked on in the past, as well as stuff I'm working on right now. Recently, I've been focusing on a couple projects leveraging the Apache Spark suite for applications in hockey analytics.</p>
+              <p>I'm hoping to be able to share some projects here that I've worked on in the past, as well as stuff
+                I'm working on right now. Recently, I've been focusing on a couple projects leveraging the Apache Spark
```
Comment (cshen98) [inline index.html]: leave text content as single line

**Review by ca-shen98 — CHANGES_REQUESTED** [review]

File: css/noscript.css [hunk 8]
```diff
@@ -0,0 +1,3 @@
+.onlyWithJS {
```
Comment (cshen98) [inline css/noscript.css]: follow css class naming conventions used: dashes, all lowercase i think

**Review by ghost — COMMENTED** [review]
address suggestions @cshen98 how does this look

**Pull request closed without merging**